src/dashboard/src/templates/rights/rights_edit.html

```diff
@@ -13,43 +14,56 @@
   <div class="row">
     <div class="col-md-12">
 
+      {% trans "Ingest" as ingest_label %}
+      {% trans "Rights" as rights_label %}
+      {% trans "Transfer" as transfer_label %}
+      {% trans "Edit" as edit_label %}
+      {% trans "Add" as add_label %}
+
       <ul class="breadcrumb">
         {% if section == 'ingest' %}
-          {% breadcrumb_url 'Ingest' 'components.ingest.views.ingest_grid' %}
+          {% breadcrumb_url ingest_label 'components.ingest.views.ingest_grid' %}
           {% breadcrumb_url name 'components.unit.views.detail' 'ingest' uuid %}
-          {% breadcrumb_url 'Rights' 'components.rights.views.ingest_rights_list' uuid %}
+          {% breadcrumb_url rights_label 'components.rights.views.ingest_rights_list' uuid %}
         {% else %}
-          {% breadcrumb_url 'Transfer' 'components.transfer.views.grid' %}
+          {% breadcrumb_url transfer_label 'components.transfer.views.grid' %}
           {% breadcrumb_url name 'components.unit.views.detail' 'transfer' uuid %}
-          {% breadcrumb_url 'Rights' 'components.rights.views.transfer_rights_list' uuid %}
+          {% breadcrumb_url rights_label 'components.rights.views.transfer_rights_list' uuid %}
         {% endif %}
         {% if id %}
-          {% breadcrumb 'Edit' %}
+          {% breadcrumb edit_label %}
         {% else %}
-          {% breadcrumb 'Add' %}
+          {% breadcrumb add_label %}
         {% endif %}
       </ul>
 
-      <h1>Rights<br /><small>{{ name }}</small></h1>
+      <h1>
+        {{ rights_label }}<br />
+        <small>{{ name }}</small>
+      </h1>
 
       {% if request.method == 'POST' %}
         {% if form.is_valid %}
           <div class="alert alert-success">
-            <p><strong>Saved!</p>
+            <p><strong>{% trans "Saved" %}</strong></p>
           </div>
         {% else %}
           <div class="alert alert-error">
-            <p>An error has occurred.</p>
+            <p>{% trans "An error has occurred." %}</p>
           </div>
         {% endif %}
       {% endif %}
 
       {% if new_content_type_created %}
           <div class="alert alert-info">
             {% if new_content_type_created == 'Other' %}
-              <p>New content added.</p>
+              <p>{% trans "New content added." %}</p>
             {% else %}
-              <p>New {{ new_content_type_created|lower }} content added.</p>
+              <p>
+                {% blocktrans with type=new_content_type_created|lower %}
```

Comment:
Is `new_content_type_created` already translated? If so, should it be `|lower`, since that may not apply to the language (or in German where I think all nouns have caps)?